### DISCUSSION TITLE
Playback not working on Chrome

### DISCUSSION THREAD

**Discussion by Guilhermerrrr** [chuot/rdio-scanner#344] [Q&A]
Hello, I'm experiencing a problem, playback is not working when using Google Chrome, mobile or on PC. The log is showing a new listener is connected and the "live feed" LED is on but no call play. I also noted the scanner title is not showing, and the buttons don't work (like opening past calls or selecting the TGs).
Am I doing something wrong or made a mistake configuring the server?
I've tested on other phones and computers with the same result, and I have no problems using Firefox (phone or computer) and Edge. 
edit: I forgot to mention that I have tested with and without audio conversion (FFMPEG is working correctly)

**Comment by chuot** [comment]
If you activate the developper console on Chrome (press F12 for that), is there any error messages on the console?

**Reply by Guilhermerrrr** [reply]
Ok, I'm using Cloudflare Tunnel and it may be the cause, but I'm truly confused why this is happening. I opened Chrome earlier today using the developer console and I got this error:

![Captura de tela 2023-04-17 11063622](https://user-images.githubusercontent.com/92966143/232547032-d123e89f-add9-48c7-9f30-6464369827cf.png)

But I tested it again now, to write you this reply and I got no errors (but no playback either). This is where I stand:
| Header | Domain name| Domain name (incognito) | Local IP | Local IP (incognito) |
|--------|--------|--------|--------|--------|
| Chrome PC| :x: | :x: | :white_check_mark: | :white_check_mark: |
| Chrome Phone | :x: | :x: | :white_check_mark: | :white_check_mark: |
| Firefox PC | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
| Firefox Phone | :white_check_mark: | :white_check_mark: | :white_check_mark: | :white_check_mark: |
| Edge | :x: | :x: | :white_check_mark: | :white_check_mark: | 

I'm very confused because if the problem was Cloudflares's tunnel I wouldn't be able to use the website using Firefox. And if the problem was with Chrome (extensions) then I would be able to use the mobile app because it doesn't contain extensions :confused:
